WISE-Community/WISE
# Styled the contact form

Styled the contact form and removed the contact complete component. (Completion confirmation message is now shown in the contact form component itself).

To test:
- Load `/contact` and test out the form to make sure it works correctly.

Closes #1587.
